.github/workflows/publish.yml

```diff
@@ -78,12 +78,10 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: '14.x'
+          node-version: "12.x"
```

Comment:
Whole ecosystem is moving towards latest LTS so we should also keep up to date and match with node 16.x when possible.